plugins/header_rewrite/matcher.h

```diff
@@ -187,7 +187,7 @@ template <class T> class Matchers : public Matcher
   }
 
   bool
-  test_reg(const std::string &t, bool nocase = false) const
+  test_reg(const std::string &t) const
```

Comment:
There is `_nocase` member flag which is used internally and the `nocase` parameter is completely ignored here as well in the `Matchers::test` function where the `test_reg` function is called.
That's why I got the liberty to remove this parameter. 
Was this the right decision?

Reply:
It looks like the API is designed so that `Matchers::set(data, COND_NOCASE)` should be called to enable that option. I think removing the parameter is was a good decision.